Changed by Stefan Seyfried <seife@novell.slipkontur.de> 2013-03-05

Just effectively skip the automake version check. We have new enough
automake in all dstributions. The default of REQUIRED_AUTOMAKE_VERSION
is 1.9 further up in the script.

Index: autogen.sh
===================================================================
--- autogen.sh
+++ autogen.sh
@@ -263,11 +263,12 @@ case $REQUIRED_AUTOMAKE_VERSION in
     1.4*) automake_progs="automake-1.4" ;;
     1.5*) automake_progs="automake-1.5 automake-1.6 automake-1.7 automake-1.8 automake-1.9 automake-1.10 automake-1.11" ;;
     1.6*) automake_progs="automake-1.6 automake-1.7 automake-1.8 automake-1.9 automake-1.10 automake-1.11" ;;
     1.7*) automake_progs="automake-1.7 automake-1.8 automake-1.9 automake-1.10 automake-1.11" ;;
     1.8*) automake_progs="automake-1.8 automake-1.9 automake-1.10 automake-1.11" ;;
-    1.9*) automake_progs="automake-1.9 automake-1.10 automake-1.11" ;;
+    # the default set above is 1.9, so just add generic automake since we dont have older stuff anyway
+    1.9*) automake_progs="automake-1.9 automake-1.10 automake-1.11 automake" ;;
    1.10*) automake_progs="automake-1.10 automake-1.11" ;;
    1.11*) automake_progs="automake-1.11" ;;
 esac
 version_check automake AUTOMAKE "$automake_progs" $REQUIRED_AUTOMAKE_VERSION \
     "http://ftp.gnu.org/pub/gnu/automake/automake-$REQUIRED_AUTOMAKE_VERSION.tar.gz" || DIE=1
